core/src/main/resources/hudson/model/AbstractProject/configure-common.jelly

```diff
@@ -28,37 +28,39 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project" xmlns:this="this">
-  <p:config-disableBuild/>
-  <p:config-concurrentBuild />
+  <div class="jenkins-section">
```

Comment:
Markup in here or somewhere similar has changed enough to break the tab bar on the job configure screen:

![image](https://user-images.githubusercontent.com/21194782/142162088-552f2da4-d101-4f29-9c74-271d69ec51f3.png)

Reply:
Fixed :) 

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/43062514/143932515-ea28dbd0-d0f2-4336-afb9-8ebc86d6639e.png">

(This actual page will be updated/replaced as seen in https://www.youtube.com/watch?v=SxSObgwjwt8)